packages/model-viewer/src/three-components/Renderer.ts

```diff
@@ -55,15 +58,31 @@ const DEFAULT_LAST_STEP = 3;
  * the texture.
  */
 export class Renderer extends EventDispatcher {
-  private static _singleton = new Renderer({debug: isDebugMode()});
+  private static _powerPreference = DEFAULT_POWER_PREFERENCE;
+  private static _singleton : Renderer;
+
+  static set powerPreference(powerPreferenceOption:string) {
+    this._powerPreference = powerPreferenceOption || DEFAULT_POWER_PREFERENCE;
+  }
+  
+  static get powerPreference() {
+    return this._powerPreference;
+  }
+
+  static getDefaultPowerPreference() : string {
+    return DEFAULT_POWER_PREFERENCE;
+  }
```

Comment:
you can just `export const DEFAULT_POWER_PREFERNCE` instead of making a getter.

Reply:
ah ok, that makes sense, I was wondering how to access it.